### ISSUE TITLE
Proposal: Clarify purpose of repo and steps to organize

### ISSUE THREAD

**Issue by jayqi** [invinst/chicago-police-data#37]
## Main idea

Go back to the roots of this repo. From the README:

>  This is a living repository of public data about Chicago’s police officers and their encounters with the public. 

This means separating out:
- The data (this repo)
- Usage of the data (keep elsewhere)
### Define a clear purpose of this repo
- A copy of the data for anybody (not just InvInst and ChiHackNight people) who wants to use it for something. This is both cleaned data and raw data, as well as supporting primary documents like FOIA requests and responses.
- A resource to facilitate people using the data. This means documentation of data, background and context, information about CPD/IPRA processes. This probably should be stuff in the wiki, and can be evolved as needed. 
- A central hub for community of data users to announce and coordinate their projects
### Keeping projects using data elsewhere

We've ended up with a whole bunch of threads of people investigating different things. It doesn't make sense for those to be merged into this repo, but we still want some way keep track of that, facilitate their work, coordinate, and potentially build a community.

Basic idea: projects or investigations using the data are separate, whether independent GitHub repo, Google docs, etc. This would include stuff like, e.g., mapping of incidents; resolving officer identities; analyzing pdf reports. Still encourage people to announce and document their projects so that everyone can know what they're working on. 
## What we need to do
- Make sure we have all the baseline dataset files, e.g., make sure we have a Cleaned April, and a system for keeping an up-to-date scrape of June IPRA Portal
- Clean up stuff in repo so we only have what any random person interested in using the data would need
- Document what every file is. Document what each dataset contains
- Startup guide for people for people to get started, potentially with examples. Great idea by Evan on Slack.
- Design a process for data users to interact with each other and with InvInst. This needs to at least include ways to (1) ask clarifying questions about the data or about CPD/IPRA, (2) post info about their individual projects so that other people can get involved, and (3) raise issues about the repo or wiki.  
### Idea: Use issue tracker as coordination tool

We're already kind of doing this, but here is a proposal to make it more structured using the tag system. We can have three tags:
- **question** -- people can ask a question about the data or about CPD/IPRA, e.g. "Why do POs have `INVOLVED_OFFICER_TYPE` set to `Victim`?" For useful general questions we can add documentation to the wiki when the answer is figured out. 
- **project** -- people can document a project they are working on. Can link to wherever they are hosting their project. This repo would only host a link and some basic info. Inspired by new ChiHackNight breakout group tracker. 
- **repo issue** -- people can raise an issue about the repo, whether with the data or the wiki. e.g., "IPRA portal scrape is missing CRID 0000000 that is on the website"
### Open question: Role of ChiHackNight group going forward

My idea here is as follows:

In short term:
- Primary work is to implement the above.

In long term:
- Once above work is done, then our group is done.
- Still potentially value in keeping a general police accountability breakout group to facilitate ChiHackNight attendees who want to use this data for something, e.g., Chaclyn visits and answers questions; tutorial of datasets; mob programming session with data. Maybe less frequently than weekly if we don't need weekly.
- If someone has a project idea that would want multiple people to contribute, can start new breakout group for that project specifically
### Open question: Messaging platform (Slack or otherwise or none)

Do we need/want a messaging platform to support a potential community of data users?

Right now, contributors from ChiHackNight are guests on InvInst's Slack. Once this effort is finished, we probably wouldn't need it for that purpose anymore. Presumably, we would want to generally keep chat for independent projects separate (i.e., not open up InvInst's Slack to anybody random person from anywhere). 

We could make a separate Slack. We could use a competitor product like [gitter](https://gitter.im) which has free public channels viewable to anyone and only requires a GitHub account to say something (advantages over Slack). Or we could decide that the Issues Tracker is sufficient for discussion and not have anything.

**Comment by evanwsun** [comment]
One of the ways that I've structured other repo issues to make it more suitable for projects is using the milestones feature of github. Each project got its own milestone and that made it much easier to compartmentalize the projects. I also think that waffle.io could be useful for something like. Waffle.io is basically a front end for your issues. You can see an example board here https://waffle.io/waffleio/waffle.io . Notice how anyone can add issues through this. Maybe this is a good way to let newcomers/journalists/other curious parties to ask questions without having to get involved in the nuances of the project. 

The problem I'm seeing with Invinst's purpose at ChiHackNight is the problem I'm seeing with a lot of the breakout groups. Mainly, that it takes a lot of effort to introduce newcomers into projects because of the background required to catch up. With that in mind, I'm not sure what the best use of our time at ChiHackNight is. Perhaps it's just a time for all of us to be working at once.

I'm going to start working on the Startup guide. Hopefully, this will give us an easy way to get newcomers acclimated fairly fast.

**Comment by alexsoble** [comment]
@jayqi I think this is a fantastic plan for the repo. I like the repo-as-data-source concept, with issues linking out to projects that use the data. 

Does this sound like a good plan going forward @rajivsinclair? 

Beyond being a good idea in and of itself, I think this is a good example of what a self-organizing / thoughtful / forward-looking group of volunteers this is. 😊

**Comment by rajivsinclair** [comment]
Thank you for writing this up so clearly! As @alexsoble has said, this is a great example of how thoughtful and forward-looking this community is. We [**Invisible Institute**] are happiest in this role as a hub/home for all the community/groups that care about making these datasets useful.

I agree with the “clear purpose of this repo” identified here. I think it’s a good plan for us to tidy things up by making separate repo(s) for investigations and other side-projects vs keeping the raw+cleaned data here. Some of this conflation emerges from work that began as a part of the process of cleaning the data to make it more useful. If you’d like to create new related repos on GitHub here in the InvInst organization, then you’re more than welcome to (just let me know if you need more privileges). This probably makes sense for some/most but maybe not all side-projects.

We created our **Slack** with the intention that it should be a meeting place for everyone who is working with us and with these datasets, so you should all feel at home/welcome there and there’s no problem with inviting others to join the conversation or to create new channels on it for ongoing sidebars or for talking about side-projects. You all are not “guests” but rather full members on [invisibleinstitute.slack.com](invisibleinstitute.slack.com) and there are no secrets that need protecting. (fyi Slack does not charge any fees for non-profit teams.) 

If **gitter** is functionally preferable then we could migrate over to a single public channel there, but I think that we’re probably better off staying together on the existing [invisibleinstitute.slack.com](invisibleinstitute.slack.com) because I worry about unnecessarily fragmenting the community and I dot want us to add the barrier of a second new thing for people to sign up for and receive notifications from, plus I like the ability to have a few/several separate channels related to our work together in one place on our Slack community.

In general, I think we’re usually best served by asynchronous communication because it encourages more fully thought-out writing, but I like to have Slack as a meeting place. **GitHub Issues** combined with **waffle.io** seems like a good solution to me, although I do worry a bit about the per-repo nature of it when some of these conversations can touch on issues that span across multiple repos. I’m not sure what the best solution to this question is – any thoughts? As much as possible, I want to avoid creating dilemmas over _“where should I put this”_ because that is no fun for anyone.

**Comment by evanwsun** [comment]
@rajivsinclair You can add multiple repos to a waffle board

**Comment by alexsoble** [comment]
@rajivsinclair ❤️ ❤️ ❤️
